src/librustdoc/html/render.rs

```diff
@@ -2496,16 +2496,48 @@ fn item_enum(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
     Ok(())
 }
 
+fn attribute_without_value(s: &str) -> bool {
+    vec!("must_use", "no_mangle", "unsafe_destructor_blind_to_params").iter().any(|x| x == &s)
+}
+
+fn attribute_with_value(s: &str) -> bool {
+    vec!("export_name", "lang", "link_section", "must_use").iter().any(|x| x == &s)
+}
+
+fn attribute_with_values(s: &str) -> bool {
+    vec!("repr").iter().any(|x| x == &s)
+}
+
+fn render_attribute(attr: &clean::Attribute, recurse: bool) -> String {
```

Comment:
Returning `Option<String>` seems appropriate instead of possibly returning an empty string